jp/components/themes/theming-demo.md

```diff
@@ -0,0 +1,335 @@
+        @include igx-dialog($light-dialog);
+        @include igx-grid-filtering($light-grid-filtering);
+        @include igx-button-group($light-button-group);
+      }
+    }
+    ...
+  }
+```
+
+<div class="divider--half"></div>
+
+- �z�X�g�v�f�� `class` �𑀍삷��֐���g���K�[����C�x���g��ݒ肵�܂��B
+
+```html
+<!-- Adding igxButton that triggers a dropdown with the theme options -->
+<div class = "grid_wrapper" igxOverlayOutlet #outlet>
+    <div class="grid-options" igxLayout igxLayoutDir="row" igxLayoutWrap="wrap">
+        <button igxButton="raised" class="addProdBtn" (click)="openDialog()">Add New Product</button>
+        <button class="drop-down-button" igxButton="raised" (click)="toggleDropDown($event, dropdown)">Themes</button>
+        <igx-drop-down #dropdown class="theme-chooser">
+            <igx-drop-down-item class="light-theme-option" [igxDropDownItemNavigation]="dropdown" (click)="selectTheme(THEME.LIGHT)">
+            </igx-drop-down-item>
+            <igx-drop-down-item class="dark-theme-option" [igxDropDownItemNavigation]="dropdown" (click)="selectTheme(THEME.DARK)">
+            </igx-drop-down-item>
+            <igx-drop-down-item class="black-theme-option" [igxDropDownItemNavigation]="dropdown" (click)="selectTheme(THEME.BLACK)">
+            </igx-drop-down-item>
+        </igx-drop-down>
+    </div>
+...
+</div>
+```
+
+<div class="divider--half"></div>
+
+**�e�[�} �N���X** ��\���f�[�^����уz�X�g�v�f�� `class` �Z���N�^�[�𑀍삷��֐���ǉ���܂��B
+
+```typescript
+
+// Enumeration that will represent the themes classes
+
+export enum THEME {
+    LIGHT = "light-theme",
+    DARK = "dark-theme",
+    BLACK = "black-theme"
+}
+
+@Component({
+    selector: "app-theme-chooser-sample",
+    styleUrls: ["./theme-chooser-sample.component.scss"],
+    templateUrl: "./theme-chooser-sample.component.html"
+})
+export class ThemeChooserSampleComponent implements OnInit {
+    // Provide a reference to the themes classes
+    public THEME: typeof THEME = THEME;
+    // Bind the host element class with the themes class and set a default value
+    @HostBinding("class")
+    public themesClass: THEME = THEME.LIGHT;
+    ...
+    // Provide a function that manipulates the the class reference
+    public selectTheme(theme: THEME) {
+        this.themesClass = theme;
+    }
+    ...
+
+```
+<div class="divider--half"></div>
+
+`click` �C�x���g�݂̂Œ�`�ς݃e�[�}��ȒP�ɕύX�ł��܂��B
+
+<div class="sample-container" style="height: 650px">
+    <iframe id="theme-chooser-iframe" seamless width="100%" height="100%" frameborder="0" src="{environment:demosBaseUrl}/theme-chooser" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="theme-chooser-iframe" data-demos-base-url="{environment:demosBaseUrl}"> stackblitz �ŊJ��
+    </button>
+</div>
+
+### ���̑��̃��\�[�X
+<div class="divider--half"></div>
+
+* [Global Theme](global-theme.md)
```

Comment:
Some of these can be localized.  We've been leaving control names in English, as I mentioned.  However, check the TOC.  "Component Themes", "Color Palette", and the grid features "Paging" and "Filtering" can be localized.  I think we leave the others in English.